admin/controllers/tech-support.js

```diff
@@ -507,6 +508,49 @@ const controller = {
     } catch (error) {
       return next(error)
     }
+  },
+
+  getPsReportLogs: async function getPsReportLogs (req, res, next) {
```

Comment:
`getPsReportLogs`: suggest we rename this to `getPsReportFoldersPage`, or perhaps, `getPsReportFolders` if you think the `Page` suffix is not useful.